src/controllers/userProfileSettings.js

```diff
@@ -0,0 +1,50 @@
+import User from '../models/user.model.js';
+import redisClient from '../helpers/redis.js';
+
+/* eslint-disable camelcase */
+const UpdateUserInfo = async (req, res) => {
+  const userId = req.user.id;
+  const {
+    gender,
+    email,
+    username,
+    password,
```

Comment:
Why did you select email, username and password? I think you don't need any if them in this case.

Reply:
No! just check well down I am using them though, I am not editing them